charts/victoria-metrics-auth/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if eq .Values.configMap "" }}
+apiVersion: v1
+kind: ConfigMap
```

Comment:
replace with secret ? helm has b64enc func, maybe it will be useful.

Reply:
Thanks @f41gh7. I changed for secret in next commit.